.env.example

```diff
@@ -1,7 +1,7 @@
 # Assumes the Postgres server was started using the Compose file in docker/postgres/docker-compose.yml
 DATABASE_URL=postgres://postgres:@localhost:5432/compass
 
-NEXTAUTH_SECRET=example-secret # Generate using `openssl rand -base64 32`
+NEXTAUTH_SECRET=  # Generate using `openssl rand -base64 32`
```

Comment:
I think a default value here is helpful for dev environments to suppress the warning?

Reply:
The main concern I have with a default value is that it might get pushed to prod accidentally. IMHO, for security related things, it is better to have an error and slight inconvenience for the developer against the risk of accidental prod push.

Reply:
I disagree but not a big deal